test/kubemark/resources/manifests/kube-addon-manager.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   hostNetwork: true
   containers:
   - name: kube-addon-manager
-    image: {{kube_docker_registry}}/kube-addon-manager:v9.1.1
+    image: {{kube_docker_registry}}/kube-addon-manager:v9.1.2
```

Comment:
do we want 9.1.2 or the current 9.1.3?

Reply:
looks like 9.1.2 is the latest in prod: https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-addon-manager/images.yaml

Reply:
oh. uh... ok... cc @spencer-p re: https://github.com/kubernetes/kubernetes/pull/96969#issuecomment-776861644